src/policy/rbf.cpp

```diff
@@ -43,3 +47,31 @@ RBFTransactionState IsRBFOptInEmptyMempool(const CTransaction& tx)
     return SignalsOptInRBF(tx) ? RBFTransactionState::REPLACEABLE_BIP125 : RBFTransactionState::UNKNOWN;
 }
 
+bool GetEntriesForRBF(const CTransaction& tx, CTxMemPool& m_pool,
+                      const CTxMemPool::setEntries setIterConflicting,
```

Comment:
I guess for performance reasons, we should pass `setIterConflicting` by reference, rather than by value?
```suggestion
                      const CTxMemPool::setEntries& setIterConflicting,
```

Reply:
Good point, done!